specificatie/naam.yaml

```diff
@@ -0,0 +1,167 @@
+openapi: 3.0.3
+info:
+  title: Naam definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    Aanhef:
+      type: string
+      description: |
+        Kun je gebruiken als aanhef in een brief gericht aan persoon.
+      example: "Hoogwelgeboren heer"
+    Aanschrijfwijze:
+      type: string
+      description: |
+        Samengestelde naam die je kunt gebruiken in de communicatie met de persoon.
+      example: "H.W. baron van den Aedel"
+    GebruikInLopendeTekst:
+      type: string
+      description: |
+        Naam van persoon die je kunt gebruiken als je in lopende tekst (bijvoorbeeld in een brief) aan persoon refereert.
+      example: "baron Van den Aedel"
+    Geslachtsnaam:
+      type: string
+      maxLength: 200
+      description: |
+        De achternaam van een persoon.
+      example: "Vries"
+    RegelVoorafgaandAanAanschrijfwijze:
+      type: string
+      description: |
+        Deze regel moet als aparte regel boven de aanschrijfwijze worden geplaatst. Komt alleen voor bij personen met een adellijke titel of predicaat.
+      example: "De hoogwelgeboren heer"
+    VolledigeNaam:
```

Comment:
@CathyDingemanse kan jij een mooie beschrijving bedenken voor volledige naam?

Reply:
Combinatie van voornamen, voorvoegsels en geslachtsnaam, zonder predicaat, adellijke titel of toevoeging van een eventuele partnernaam.

Reply:
done

Reply:
@CathyDingemanse in de volledigeNaam wordt de adellijke titel of predikaat wel toegevoegd, alleen niet de partnernaam.

Is er een mooie term of omschrijving om aan te geven dat dit de wettelijke naam, geregistreerde naam of iets dergelijks is? Om het verschil met de tegenstelling tot de aanschrijfnaam duidelijk te maken?
Dus kunnen we in de omschrijving zeggen wat/waarvoor deze is i.p.v. hoe we deze samenstellen?

Reply:
@MelvLee @CathyDingemanse de description klopt nu niet. Voorlopig hiervan maken (tot we iets beters verzinnen): "Combinatie van predicaat, voornamen, adellijke titel, voorvoegsels en geslachtsnaam, zonder toevoeging van een eventuele partnernaam."?

Reply:
done